view-user/src/components/common/BingoResult/BingoResult.module.css

```
@@ -0,0 +1,53 @@
+.container {
+    /* height: 100vh; */
+    width: 100vw;
+    background-color: #C9AAAA;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .list_container {
+    height: 90vh;
+    width: 90vw;
+  }
+  
+  .card_frame {
+    border-radius: 3px;
+    border-width: 0px;
+    border-style: solid;
+    border-color: #333;
+    padding: 2rem;
+    background-color: #D9D9D9;
+  }
+
+  .card_text {
+    font-size: 3rem;
+    font-weight: bold;
+    color: #333;
+    padding-bottom: 10px;
+    padding-left: 10px;
+  }
+  
+  .card {
+    display: grid;
+    grid-template-columns: repeat(8, 1fr);
+    grid-gap: 10px;
+  }
+  
+  .card_content {
+    width: 150px;
+    height: 150px;
```

```suggestion
    width: 100px;
    height: 100px;
```
ちょっと小さくしてぱっと見で見れる量増やしてもいいかも。私の環境だとこの設定で10個並んだ